lib/rubocop/cop/style/invertible_unless_condition.rb

```diff
@@ -99,23 +99,23 @@ def preferred_condition(node)
           end
         end
 
-        def preferred_send_condition(node)
-          receiver_source = node.receiver.source
+        def preferred_send_condition(node) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I didn't see a good way to avoid disabling this cop. It would require breaking apart this method, but I don't know that that would actually improve understandability.

Open to suggestions though.

Reply:
Another option might be to explicitly fallback to `self` as the `receiver_source` and then leave all the rest the same, and rely on the redundant self correction to clean it up, but I don't know if that would actually fix the complexity offense, as it still introduces the conditional, even if it's inline.